### PR TITLE
[C++ / en] Added Tuple.

### PR DESCRIPTION
**Pull request by Jaskamalkainth**

### CODE REVIEW

File: c++.html.markdown [hunk 1]
```diff
@@ -955,6 +955,53 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
...
+///////////////////////////////////////
+
+#include<tuple>
+
+// Conceptually, Tuples are similar to  old data structures (C-like structs) but instead of having named data members , 
+// its elements are accessed by their order in the tuple.
+
+// We start with constructing a tuple.
+// Packing values into tuple
+auto first = make_tuple( 10 , 'A' ) ;
```
Comment (vendethiel) [inline c++.html.markdown]: Doesn't seem fixed? Maybe it's the wrong branch?

Comment (Jaskamalkainth) [inline c++.html.markdown]: Oh! I will send PR again after removing all the spaces. 
I gave spaces just to make  code more Readable and Beautiful.

File: c++.html.markdown [hunk 2]
```diff
@@ -955,6 +955,53 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
...
+///////////////////////////////////////
+
+#include<tuple>
+
+// Conceptually, Tuples are similar to  old data structures (C-like structs) but instead of having named data members , 
+// its elements are accessed by their order in the tuple.
+
+// We start with constructing a tuple.
+// Packing values into tuple
+auto first = make_tuple(10,'A') ;
```
Comment (vendethiel) [inline c++.html.markdown]: still spaces before the `;`.

Please, look at the rest of the file, and follow the style used :).

Comment (Jaskamalkainth) [inline c++.html.markdown]: Sorry! Fixed! :)

**Comment by vendethiel** [comment]
Thanks! :cake:

**Comment by vendethiel** [comment]
I went ahead and polished the coding style in e1016455d5e4472e7a533c8cdd6df8ae4f2e7854

**Pull request merged**